src/main.cpp

```diff
@@ -179,6 +201,14 @@ void slice(int argc, char **argv)
                     case 'v':
                         cura::increaseVerboseLevel();
                         break;
+#ifdef _OPENMP
+                    case 'm':
+                        str++;
+                        n_threads = *str - '0';
+                        n_threads = std::max(1, n_threads);
+                        omp_set_num_threads(n_threads);
+                        break;
```

Comment:
It's not just 3 lines. It's all of the parameter parsing really. But I understand that it's a lot of work. Just to say that I think a refactor is eventually needed.